prbt_hardware_support/test/unittests/unittest_modbus_adapter_operation_mode.cpp

```diff
@@ -49,6 +49,9 @@ class ModbusAdapterOperationModeTest : public testing::Test
 public:
   ModbusAdapterOperationModeTest();
 
+  virtual void SetUp() override;
+  virtual void TearDown() override;
```

Comment:
Same as above. Why not use constructor/destructor?

Reply:
Done.